SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -724,9 +715,7 @@ chk_for_maintenance() {
     local cmaint="true"
     # check maintenance mode of cluster
     # msl resource is reported unmanaged, but cln resource not
-    if crm_mon -r1 | grep "Resource management is DISABLED" >/dev/null 2>&1; then
-        cluInMaint="true"
-    fi
+    cluInMaint=$(cibadmin -Q | xmllint -xpath "string(//crm_config/*/nvpair[@name='maintenance-mode']/@value)" -)
```

Comment:
+1 - yes this one lets disappear my fake resource with fake-parameter maintenance-mode="fake" from the answer-list :)